.github/workflows/ci.yml

```diff
@@ -330,15 +330,12 @@ jobs:
       - name: Install packages
         if: matrix.features == 'huge'
         run: |
-          brew install lua libtool
-          echo "LUA_PREFIX=$(brew --prefix)" >> $GITHUB_ENV
-
-      - name: Install blackhole-2ch for macos-12
-        if: matrix.features == 'huge' && matrix.runner == 'macos-12'
-        run: |
-          # Install audio device for playing sound since some of macos-12 machines have no audio device installed.
-          if system_profiler -json SPAudioDataType | jq -er '.SPAudioDataType[]._items == []'; then
-            brew install blackhole-2ch
+          if [ $(uname -r | grep ^21 | cut -c1-2) ]; then
```

Comment:
what is the output of this uname command chain? Can you please add a comment for when we need to install lua and blackhole-2ch?

Reply:
> what is the output of this uname command chain?

oh well I missed something:
`if [ $(uname -r | grep ^21 | cut -c1-2) == 21 ]`
Identifies macos-12

> Can you please add a comment for when we need to install lua and blackhole-2ch?

I will add the comments back was thing about it too.